README.md

```diff
@@ -12,65 +12,86 @@
 
 ## Installation
 
-Install the package and all dependencies with: 
+Install the package and all dependencies with:
 
 ```r
 remotes::install_github("epiforecasts/NCoVUtils", dependencies = TRUE)
 ```
 
 ## Usage
 
-See the individual function documentation for full details of present functionality. The main function imports and cleans the linelist. 
+### Worldwide data
 
-```r
-NCoVUtils::get_linelist()
-```
+There are three sources of worldwide, country-level data on cases and deaths.
 
-Extract international case counts compiled by WHO using:
+1. Extract total global cases and deaths by country, and specify source, using:
+  + ```NCoVUtils::get_total_cases(source = c("WHO", "ECDC"))```
+2. Extract daily international case and death counts compiled by the WHO using:
+  + ```NCoVUtils::get_who_cases(country = NULL, daily = TRUE))```
+3. Extract daily international case and death counts compiled by ECDC using:
+  + ```NCoVUtils::get_ecdc_cases()```
 
-```r
-NCoVUtils::get_who_cases()
-```
+A further function for worldwide data extracts non-pharmaceutical interventions by country:
 
-Extract international case counts compiled by ECDC using:
+* ```NCoVUtils::get_interventions_data()```
 
-```r
-NCoVUtils::get_ecdc_cases()
-```
+And anonymised international patient linelist data can be imported and cleaned with:
 
-Extract total global cases (and deaths) by country using:
+* ```NCoVUtils::get_linelist()```
 
-```r
-NCoVUtils::get_total_cases()
-```
+### Sub-national data
 
-Extract non-pharmaceutical interventions by country using:
+We have several functions to extract sub-national level data by country. These are typically at the admin-1 level, the largest regions available. We are also working on joining the data to standard georeferencing codes to allow easy mapping.
 
-```r
-NCoVUtils::get_interventions_data()
-```
+Currently we include functions for sub-national data in the following countries:
+
+Europe
+
+  +	Belgium
+
+  +	France
+
+  + Germany
+
+  +	Italy
+
+  +	Spain
+
+  + United Kingdom
+
+Americas
+
+  +	Canada
+
+  +	United States
+
+Eastern Mediterranean
+
+  + Afghanistan
+
+Western Pacific
+
+  + Korea
+
+  + Japan
+
+South-East Asia
+
+  + None currently available
+
+Africa
+
+  + None currently available
 
-Extract country-level regional cases from:  
 
-* Belgium  
-* Canada  
-* France  
-* Germany  
-* Italy  
-* Spain  
-* United Kingdom  
-* United States  
-* Japan  
-* Korea
-* Afghanistan
+We are working to include more countries.
 
-See the function reference for a full list of data sets that package can be used to extract.
 
 ## Development
```

Comment:
maybe add sentence that help is appreciated, see issues? or describe some domains where help is appreciated (new data sources, geo-coding, ....)?